ontology/yaml/resources/HVAC/entity_types/VAV.yaml

```diff
@@ -388,6 +388,13 @@ VAV_DSP_ED:
   - DSP
   - ED
 
+VAV_RDM:
```

Comment:
This device does not have flow control? How does it know what to set the damper to? What does it control to? It should have control.

Reply:
Actually, there are two VAV, one at the supply side that control the damper regarding the CO2 concentration. And this one is the return that copy the supply damper command in order to extract the same amount of air that the supply bring.

Reply:
ok. please rename RDM to RDC and its good to go.